README.md

```diff
@@ -260,55 +263,23 @@ agent/client health information/inventory metadata to downstream exporters.
 
 ### <a name="collector-usage"></a>Usage
 
-First, install the collector if you haven't.
+The collector is in its initial development stages. It can be run directly
+from sources, binary, or a Docker image.
 
+1. Run from sources:
 ```
-$ go get github.com/census-instrumentation/opencensus-service/cmd/occollector
+$ go run github.com/census-instrumentation/opencensus-service/cmd/occollector
 ```
-
-Create a config.yaml file in the current directory and modify
-it with the collector exporter configuration.
-
-config.yaml:
-
+2. Run from binary (from the root of your repo):
 ```
-omnition:
-  tenant: "your-api-key"
+$ make collector
+$ ./bin/occollector_$($GOOS)
 ```
-
-Next, install ocagent if you haven't.
-
+3. Build a Docker scratch image and use the appropria Docker command for your scenario:
 ```
-$ go get github.com/census-instrumentation/opencensus-service/cmd/ocagent
-```
-
-Create a config.yaml file in the current directory and modify
-it with the collector exporter configuration.
-
-config.yaml:
-
+$ make docker-collector
+$ docker run --rm -it -p 55678:55678 occollector
 ```
-collector:
-  endpoint: "https://collector.local"
-```
-
-Run the example application that collects traces and exports
```

Comment:
Why did you remove the stuff from here down below? They are necessary for folks to get a feel for what the agent does, they are unrelated to the collector.

Reply:
Ops cut too much

Reply:
Probably need to be moved to a section closer to the agent...

Reply:
Actually, same it is already located on top I will do a small update to it.

Reply:
Hey @pjanotti I didn't see an update to this content that was removed but it was marked resolved.

Reply:
The content at the bottom was removed since it was already present before it.

https://github.com/census-instrumentation/opencensus-service#running-an-end-to-end-exampledemo